apps/test/unit/templates/MakerLandingTest.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import MakerLanding from '@cdo/apps/templates/MakerLanding';
+
+describe('MakerLanding', () => {
+  it('renders', () => {
+    const fakeTopCourse = {
+      assignableName: 'Maker',
+      lessonName: 'Maker Lesson',
+      linkToOverview: 'fakelink.com',
+      linkToLesson: 'fakelink2.com'
+    };
+
+    shallow(<MakerLanding topCourse={fakeTopCourse} />);
+  });
```

Comment:
Is an assertion not needed here?

Reply:
Correct, the "test" in these "it renders" tests is just that it's able to render (ie, shallow doesn't error).

Reply:
Cool!  Would it make sense to add another unit test to check the presence of a notification if included?